core/controllers/editor.py

```diff
@@ -294,6 +309,8 @@ def _get_exploration_data(self, exploration_id, version=None):
             'tags': exploration.tags,
             'title': exploration.title,
             'version': exploration.version,
+            'is_version_of_draft_valid': is_version_of_draft_valid,
+            'draft_changes': draft_changes
```

Comment:
@526avijitgupta The backend is sending a copy of the draft changes to the UI. The UI can choose to use it however it wants.